src/shared/components/Checkout/Checkout.styles.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import { colors } from '@/shared/theme'
+import { Icon } from '..'
+
+export const Container = styled.div`
+  padding: 24px;
+  width: 312px;
+  background-color: ${colors.gray[800]};
+  color: ${colors.white};
+`
+
+export const ProgressbarContainer = styled.div`
+  width: 24px;
+  height: 24px;
+`
+
+export const StepsProgressContainer = styled.div`
+  display: flex;
+  gap: 12px;
```

Comment:
`gap` for flexbox isn't well supported, so maybe we could change it - https://caniuse.com/?search=gap 
ls it possible to replace it with some margins and maybe padding with no effort? What do you think?

Reply:
On the second thought, maybe I'm wrong here. Users will have to use Chrome or Firefox anyway to do all the interactions with Polkadot, so maybe it's not an issue at all.

Reply:
Yeah, I think it might not be worth replacing it because of what you mention; the user already needs to be on a modern browser, and if that's not the case, we would have to change it in several more places where is being used

Reply:
I agree with Bartosz here, the support is pretty poor (69%). Currently our target is +/- 90% so we should be mindful of that. I don't think we actually use `gap` for flexbox anywhere (we use it for grid but that's much better supported).

Reply:
I also don't think our users need to be on a _very_ modern browser, as far as I'm aware our least supported feature atm is resize observer so it's not cutting edge